schema.sql

```diff
@@ -1,5 +1,4 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+create table animals(id bigserial not null primary key, name varchar(20), date_of_birth date, escape_attempts integer, neutered boolean, weight_Kg decimal);
+alter table animals add column species varchar(20);
```

Comment:
**OPTIONAL** - This is not part of the requirement for this milestone, I suggest you include it in the next milestone. Please fix this 👍